build/helpers/coverage.js

```diff
@@ -41,7 +61,34 @@ process.on("exit", () => {
         continue;
       }
 
-      parseScript(scriptCoverage);
+      if (scriptCoverage.url !== impl) {
+        continue;
+      }
+      const script = parseScript(scriptCoverage);
+
+      if (script === null) {
+        continue;
+      }
+
+      let computedPoints = computePoints(script);
+      let sortedBlocks = script.coverage
+        .filter(block => block.count == 0)
```

Comment:
Should be `===`; I need to find a way of enabling the linter rules for JavaScript files as well.